src/planwise/model/projects2.clj

```diff
@@ -30,7 +32,14 @@
 (s/def ::capacity number?)
 (s/def ::providers (s/keys :req-un [::capacity]))
 
-(s/def ::config (s/nilable (s/keys :req-un [::demographics ::actions ::coverage ::providers])))
+(defmulti attribute-analysis-type-actions :analysis-type)
+(defmethod attribute-analysis-type-actions "budget" [_]
+  (s/keys :req-un [::analysis-type ::actions]))
```

Comment:
Since the `::analysis-type` is in `::config-base`, it should not be needed in the attribute-analysis-type-actions overloads I thinks

Reply:
I'll check and fix it.